test/WebSites/RazorWebSite/Views/ViewEngine/ViewWithRelativePath.cshtml

```diff
@@ -12,4 +12,4 @@
 }
 ViewWithRelativePath-content
 <partial>@await Html.PartialAsync("../Shared/_PartialThatSetsTitle.cshtml")</partial>
-@await Component.InvokeAsync("ComponentWithRelativePath", person)
\ No newline at end of file
+@await Component.InvokeAsync("ComponentWithRelativePath", new { person })
```

Comment:
One thing, tooling sucks with this doesn't it? Aka second you type `{` does it properly auto-complete the braces? If not we should file a tooling bug. I think I recall it being difficult to type out in the Razor editor.